src/main/java/svydovets/core/context/beanDefinition/BeanDefinitionFactory.java

```diff
@@ -139,8 +141,8 @@ private String doResolveScopeName(Scope scopeAnnotation) {
 
             throw new UnsupportedScopeException(errorMessage);
         }
-        log.trace("Scope values is not specified explicitly. Set '{}' by default", ApplicationContext.SCOPE_SINGLETON);
-        return ApplicationContext.SCOPE_SINGLETON;
+        log.trace("Scope values is not specified explicitly. Set '{}' by default", SCOPE_SINGLETON);
+        return SCOPE_SINGLETON;
```

Comment:
What do you think about the Scope enum? Is it possible to use it?

Reply:
on first iteration I added enum, but I had problem when init default value for @Scope